pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -166,19 +165,6 @@ func fixUserPartOfCrcUsersAndHypervAdminsGroup() error {
 	return errReboot
 }
 
-func checkIfHyperVVirtualSwitchExists() error {
```

Comment:
Fwiw, it would be good to split the system mode networking removal on Windows in its own preparatory commit, this would make it easier to focus on/review the config/preflight changes.

Reply:
Honestly, I do not even know if this functionality still works. I'll consider, but these checks are also performed when not even necessary (like the existence of the virtual switch) and therefore actually wrong.

Reply:
I've done this in https://github.com/cfergeau/crc/tree/libhvee, this removes even more code